be/src/vec/exec/join/join_op.h

```diff
@@ -115,7 +115,7 @@ struct RowRefList : RowRef {
     };
 
     RowRefList() {}
-    RowRefList(const Block* block_, size_t row_num_) : RowRef(block_, row_num_) {}
+    RowRefList(size_t row_num_) : RowRef(row_num_) {}
 
     ForwardIterator begin() { return ForwardIterator(this); }
```

Comment:
iterating RowRefList through ForwardIterator is slow, consider providing a interface to get all row_num list